src/packages/core/property-editor/uis/tree-picker/property-editor-ui-tree-picker.element.ts

```diff
@@ -52,6 +62,37 @@ export class UmbPropertyEditorUITreePickerElement extends UmbLitElement implemen
 		this.ignoreUserStartNodes = config?.getValueByAlias('ignoreUserStartNodes');
 	}
 
+	constructor() {
+		super();
+
+		this.#dynamicRootRepository = new UmbDynamicRootRepository(this);
+
+		this.consumeContext(UMB_WORKSPACE_CONTEXT, (workspaceContext: typeof UMB_WORKSPACE_CONTEXT.TYPE) => {
```

Comment:
This type cast of course guarantees that your build will fail if you don't get the correct type, but let's say we change the output type, you would have to also replace this line. I think it's safe enough to accept the implied type:

```suggestion
		this.consumeContext(UMB_WORKSPACE_CONTEXT, (workspaceContext) => {
```